pkg/hs/urbit-king/lib/Urbit/Arvo/Effect.hs

```diff
@@ -144,6 +144,7 @@ data Tint
     | TintK
     | TintW
     | TintNull
+    | TintTrue Atom Atom Atom
```

Comment:
I suspect the haskell thing to do is to make these a `Word8` ...

Reply:
That's right. Please do so.